packages/@react-aria/color/src/useColorField.ts

```diff
@@ -100,17 +104,17 @@ export function useColorField(
     }
   };
 
-  let {inputProps, ...otherProps} = useFormattedTextField(
-    mergeProps(props, {
-      id: inputId,
-      value: inputValue,
-      defaultValue: undefined,
-      validate: undefined,
-      [privateValidationStateProp]: state,
-      type: 'text',
-      autoComplete: 'off',
-      onChange
-    }), state, ref);
+  let {inputProps, ...otherProps} = useFormattedTextField({
+    ...props,
```

Comment:
`mergeProps` resulted in `onChange` being called twice, with both the raw text value of the field, and with the parsed color. It should only be called with a color.

Reply:
I think this has closed https://github.com/adobe/react-spectrum/issues/5484